samples/openapi3/server/petstore/spring-boot-oneof/src/main/java/org/openapitools/OpenAPI2SpringBoot.java

```diff
@@ -0,0 +1,23 @@
+package org.openapitools;
+
+import com.fasterxml.jackson.databind.Module;
+import org.openapitools.jackson.nullable.JsonNullableModule;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+
+@SpringBootApplication
+@ComponentScan(basePackages = {"org.openapitools", "org.openapitools.api" , "org.openapitools.configuration"})
+public class OpenApiGeneratorApplication {
```

Comment:
Any idea why this is being created? I wouldn't imagine this should exist from your change

Reply:
I have no idea.  I have simply executed the test script and they have been generated.  

what should I do? Remove It?

Reply:
If you regenerate on `master` does it also output these files?

Reply:
I deleted those examples and relaunched the script and it worked correctly.
It is not clear to me what happened before.